app/assets/locales/android_translatable_strings.txt

```diff
@@ -115,6 +115,7 @@ sync.fail.bad.data=Server provided improperly formatted data, please try again o
 sync.fail.bad.local=Your information was fetched, but a problem occurred with the log in. Please try again.
 sync.fail.bad.network=Couldn't contact server. Please make sure an internet connection is available or try again later.
 sync.fail.server.error=The server encountered an error processing your data. Please try again later. If the problem persists, contact CommCare Support.
+sync.fail.rate.limited.server.error=Our servers are unavailable at this time. Please try again later. If the problem persists, contact CommCare Support.

```

Comment:
I think we should remove the "contact CommCare Support" part of this message (and the one below).

Generally I think we should never in-code suggest to people that there is a "CommCare Support," but with this issue especially, a 503 error specifically is a message to the phone that there is nothing to report to support, and I don't want to increase the burden on an arbitrary support team if they are getting 503'd.